fl4health/strategies/fedpca.py

```diff
@@ -0,0 +1,150 @@
+from logging import WARNING
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+from flwr.common import (
+    EvaluateRes,
+    FitRes,
+    GetPropertiesIns,
+    MetricsAggregationFn,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.common.logger import log
+from flwr.server.client_manager import ClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy import FedAvg
+
+from fl4health.client_managers.base_sampling_manager import BaseFractionSamplingManager
+from fl4health.parameter_exchange.parameter_packer import PrincipalComponentsPacker
+from fl4health.strategies.strategy_with_poll import StrategyWithPolling
+
+
+class FedPCA(FedAvg, StrategyWithPolling):
```

Comment:
Conceptually it doesn't really make sense to inherit from `FedAvg`, but this way reduces a lot of code duplication if we wish to adhere to making FedPCA a strategy.